app.rb

```diff
@@ -78,7 +78,7 @@ def create_teacher
     print 'Specialization:'
     specialization = gets.chomp
 
-    @people << Teacher.new(age, specialization, name)
+    @people << Teacher.new(specialization, age, name)
```

Comment:
- [ ] Great job working on the refactor so far but I think it would be great if you not make the `create_student` and `create_teacher` in seprate methods since that considered as a Bad example of Single responsability rule, and also as a Bad example of YAGNI and KISS as the contents fo this method could've been placed somewhere else so please take look to Best practices and [Solid lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) ✨🚀💯

- it should be in one method you can refactor other things like the questions for asking the users to enter an input or refactor as much as you can to have clean code 💯👏🏻

![image](https://github.com/raihan2bd/school-library/assets/58553711/48810909-020e-400d-a2a9-77251e104891)